docs/modules/ROOT/pages/identity.adoc

```diff
@@ -246,32 +242,29 @@ The two certificates need to be provided to FeatureHub without their surrounding
 
  - `saml.google.sp.x509-cert` - (required) the full text of the X509 Certificate
  - `saml.google.sp.x509-cert-new` - (optional) an optional extra X509 Certificate if yours is expiring
- - `saml.google.sp.private-key` - (required) the full private key (minus comments)  
+ - `saml.google.sp.private-key` - (required) the full private key (minus comments)
 
 If you are using a properties file, you can do line continuations with a `\` character, see below for an
 example.
 
-=== What you need to tell your SAML provider you need
+=== Setting up SAML provider
 
-When you set up your SAML provider, they will ask for 
+When you set up your SAML provider, please provide the following:
 
 ==== NameID
-The NameID format - we need you to specify `email` - the NameID
-passed back must contain the user's email. 
+The NameID format - specify `email` - the NameID
+passed back must contain the user's email.
 
 ==== User Name
-Further we use standard OASIS codes for containing the Display Name and/or the First Name and Last 
-name. Either the display name *or* the First Name/Last Name 
+Further we use standard OASIS codes for containing the Display Name and/or the First Name and Last
+name. Either the display name *or* the First Name/Last Name
 must be provided, there are no other requirements.
 
-Your SAML provider will ask you what extra attributes are required, so:
+Your SAML provider will ask you what extra attributes are required:
 
  - Last Name = `urn:oid:2.5.4.4`
  - First Name = `urn:oid:2.5.4.42`
- - Display Name = `urn:oid:2.16.840.1.113730.3.1.241` 
-
-And yes, we realise these are bizarre names but they are from the days of OASIS committees where people smarter than
-us thought this was a good idea. We will reject the login if the name isn't available.
```

Comment:
You need to make it clear that either LastName/FirstName or DisplayName are required and that we will reject the login if the name isn't available (you deleted that bit)

Reply:
it already says this ```Either the display name *or* the First Name/Last Name
must be provided, there are no other requirements.```

should be enough?